lib/api.js

```diff
@@ -232,6 +239,11 @@ class Api extends Emittery {
 				}
 
 				const worker = fork(file, options, apiOptions.nodeArguments);
+				worker.onStateChange(data => {
+					if (data.type === 'test-timeout-configured' && !apiOptions.debug) {
+						customTimeout = data.period;
```

Comment:
Store the moment until which timeouts should be ignored, e.g.:

```suggestion
						ignoreTimeoutsUntil = Math.max(ignoreTimeoutsUntil, Date.now() + data.period);
```

Then in the debounced timer function above, return if `Date.now() < ignoreTimeoutsUntil`.

Reply:
Thanks for the tip. I've updated accordingly.